<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MountWidgetClass</class>
 <widget class="QDialog" name="MountWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mount partitions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="currentRow">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>/dev/sda1 (40Gb ext4), no label</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>/dev/sda2 (200Gb jfs), label: HOME</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="noteLabel">
     <property name="text">
	     <string>Selected partition will be mounted to appropriate /var/mpkgmount/dev/ directory</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
